staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -258,7 +258,7 @@ func NewKubectlCommand(o KubectlOptions) *cobra.Command {
       kubectl controls the Kubernetes cluster manager.
 
       Find more information at:
-            https://kubernetes.io/docs/reference/kubectl/overview/`),
+            https://kubernetes.io/docs/reference/kubectl/`),
```

Comment:
Late aside: https://k8s.io/docs/reference/kubectl/ also works.

Reply:
If we want to use a short URL such as https://k8s.io/kubectl-reference, file a feature request against [k/website](https://github.com/kubernetes/website) - it's easy enough to add.